### DISCUSSION TITLE
在renderer为svg的情况下，图例翻页不生效

### DISCUSSION THREAD

**Discussion by Jenna-u** [antvis/G2#5029] [Q&A] [answered]
### 在renderer为svg的情况下，图例翻页不生效
* **G2 Version**: 4.0.9
* **Platform**: mac
* **Mini Showcase(like screenshots)**:
* **CodePen Link**:
https://codepen.io/jenna-u/pen/NWxqOKv

**Comment by mturoci** [comment]
@hustcc any updates on this? Do you accept PRs?

**Comment by hustcc** [comment, marked as answer]
SVG renderer works not well in version 4.x because of  bugsof renderer engine, but we fix all bug in 5.0 version, here is a [demo](https://g2.antv.antgroup.com/examples/renderer/renderer/#svg).
